mlflow/tracking/service.py

```diff
@@ -83,6 +83,16 @@ def create_experiment(self, name, artifact_location=None):
             artifact_location=artifact_location,
         )
 
+    def delete_experiment(self, experiment_id):
+        """Deletes experiment with ID from backend store.
+        """
```

Comment:
nit: keep `"""` on previous line since it fits.

Although maybe we want to mention here that the experiment can be restored until it is permanently deleted.